src/Service/ZGWToXxllncService.php

```diff
@@ -490,108 +300,91 @@ public function mapZGWToXxllnc(string $casetypeId, ObjectEntity $zaakTypeObject,
         // Unset empty keys.
         $caseArray = array_filter($caseArray);
 
-        $sourceId = $this->sendCaseToXxllnc($caseArray, $synchronization);
-        if (!$sourceId) {
+        $sourceId = $this->sendCaseToXxllnc($caseArray, $caseObject, $synchronization);
+        if ($sourceId === false) {
+            var_dump("POST to xxllnc failed.");
```

Comment:
Shouldn't you put this in a log?

Reply:
vardump removed, sendCaseToXxllnc should throw error logs